gui/screens/settings.py

```diff
@@ -173,3 +173,33 @@ def get_community_info(self, name: str) -> UserCommunityInfo:
     def open_community_overview_settings(self, name: str):
         driver.mouseClick(self._get_community_item(name))
         return CommunitySettingsScreen().wait_until_appears()
+
+
+class KeycardSettingsView(QObject):
+
+    def __init__(self):
+        super(KeycardSettingsView, self).__init__('mainWindow_KeycardView')
+        self._scroll = Scroll('settingsContentBaseScrollView_Flickable')
+        self._setup_keycard_with_existing_account_button = Button('setupFromExistingKeycardAccount_StatusListItem')
+        self._create_new_keycard_account_button = Button('createNewKeycardAccount_StatusListItem')
+        self._import_restore_via_seed_phrase_button = Button('importRestoreKeycard_StatusListItem')
+        self._import_from_keycard_button = Button('importFromKeycard_StatusListItem')
+        self._check_whats_on_keycard_button = Button('checkWhatsNewKeycard_StatusListItem')
+        self._factory_reset_keycard_button = Button('factoryResetKeycard_StatusListItem')
+
+    @allure.step('Check that keycard screen displayed')
+    def check_keycard_screen_loaded(self):
+        assert KeycardSettingsView().is_visible
+
+    @allure.step('Check that all keycard options displayed')
+    def all_keycard_options_available(self):
+        assert self._setup_keycard_with_existing_account_button.is_visible, f'Setup keycard with existing account not visible'
+        assert self._create_new_keycard_account_button.is_visible, f'Create new keycard button not visible'
+        assert self._import_restore_via_seed_phrase_button.is_visible, f'Import and restore via seed phrase button not visible'
+        self._scroll.vertical_scroll_to(self._import_from_keycard_button)
+        assert driver.waitFor(lambda: self._import_from_keycard_button.is_visible,
+                              10000), f'Import keycard button not visible'
```

Comment:
can we replace 10000 with configs.timeouts.UI_LOAD_TIMEOUT_MSEC ?

Reply:
changed to configs.timeouts.UI_LOAD_TIMEOUT_MSEC